jekyll/_cci2/deploy/configure-deploy-markers.adoc

```diff
@@ -21,20 +21,209 @@ Deploy markers provide a lightweight way to log your deployments without requiri
+
+If you don't already have an Environment in your Organization a new one will be created with the name `default`. This will be set as the target for this deploy. See <<manage-environments>> for more information.
+
+If you don't already have a Component in your Project a new one will be created with the name of the project. This will be set as the component that is being deployed.
+
+If you have multiple environments and components you will have to specify their names manually.
+
+This can be done as shown below:
+
+[,yml]
+----
+jobs:
+  deploy-my-service:
+    executor: some-executor
+    steps:
+      - run: circleci run release plan --environment-name=<some-environment-name> --component-name=<some-component-name> --target-version=<some-version-name> --namespace=<some-namespace>
+----
 
-== 2. Update configuration
+In the example
 
-Next you will update your CircleCI configuration file. Add a `circleci run release log` command to your deployment job. This tells CircleCI to log the deploy marker in the deploys UI.
+** The `environment-name` parameter sets the target environment. If said environment doesn't exist, it will be created.
+** The `component-name` parameter sets the name that will be displayed in the CircleCI UI.
+** The `namespace` parameter is optional and can be provided to use a value other than `default`.
 
-Add a `log release` step at the end of your _deployment_ job, for example:
+If you are deploying multiple components or to multiple environments from a single workflow, you will have to provide the command with a deployment name.
+This identifies the deployment your are planning so that you can later on reference it to update its status.
+This can be done as shown below:
+
+[,yml]
+----
+jobs:
+  deploy-my-service:
+    executor: some-executor
+    steps:
+      - run: circleci run release plan <deploy-name> --environment-name=<some-environment-name> --component-name<=>some-component-name> --target-version=<some-version-name>
+----
+
+In the example the positional argument `deploy-name` is the arbitrary value that will be used to identify the deployment and should be unique within the workflow.
+If not specified the deployment name will be set to `default`.
+
+=== 1.2. Update the deploy status
+
+After deploying your application, you can update the status of the deployment to `RUNNING` by running the `circleci run release update` command in a new step.
+
+[,yml]
+----
+circleci run release update --status=running
+----
+
+If you are deploying multiple components or to multiple environments from a single workflow, you will have to provide the command with a deployment name.
+This value should match the value you provided when you planned the deploy.
+
+[,yml]
+----
+circleci run release update <deploy-name> --status=running
+----
+
+Now you can add `on_success` and `on_failure` steps at the end of your deployment job, to handle the final status update of the deploy.
+
+pass:[<!-- vale off -->]
```

Comment:
What was failing to mean you needed to bypass the linter? Just wondering because it should skip code blocks by default anyway 🤔

Reply:
you are right, I forgot to take this out. If I recall something in the formatting was messing up the code block. I will remove it.